src/github.ts

```diff
@@ -139,7 +149,13 @@ export const createGithubRelease = async (
     prerelease: !productionRelease,
     make_latest: productionRelease && PLATFORMS_FLAGGED_LATEST.includes(platform) ? 'true' : 'false',
     name: releaseName,
-    body: releaseNotes ?? (await generateReleaseNotesFromGithubEndpoint(owner, repo, appOctokit, newVersionName)),
+    body: releaseNotes,
   })
-  console.log(release.data.id)
+
+  console.log(
+    JSON.stringify({
+      id: release.data.id,
+      releaseNotes: formatReleaseNotesForMattermost(releaseNotes).replaceAll('\n', '\\n'),
```

Comment:
isn't `\\n` => `\n` because double backslash with result in `\` right?

Reply:
Not quite. `\n` is a linebreak but it needs to be escaped since JSON values are not allowed to be multiline strings. Not escaping them with `\\n` otherwise crashes parsing with e.g. `jq`.